src/modules/displayComments.js

```diff
@@ -0,0 +1,41 @@
+import countComments from './countComments.js';
+
+const displayComments = async () => {
+  const commentId = document.querySelector('.movie-title').id;
+  const url = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/IR7MovTrVQtBQVyC6UTK/comments?item_id=${commentId}`;
+
+  const commentsList = document.querySelector('.movie-comments');
+  // commentsList.innerHTML = '';
+
+  try {
+    const response = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    // const commentsCounter = document.querySelector('.comments-counter');
+
+    // if (response.error === 400) {
+    //   commentsCounter.innerHTML = '<p>Comments (0)</p>';
+    // }
```

Comment:
- [ ] Kindly remove unwanted codes completely instead of adding them as comments. It makes your code look kind of messy and prone to bugs :wink: